### PR TITLE
Added a good number of Blue Team tools

### PR DESCRIPTION
**Pull request by D3vil0p3r**

### CODE REVIEW

**Review by Livelove666 — APPROVED** [review]

**Comment by D3vil0p3r** [comment]
Thank you for approving it @Livelove666 Can you merge this PR and also the previous one https://github.com/fabacab/awesome-cybersecurity-blueteam/pull/27 where I fixed some broken link?

Thank you in advance

**Review by fabacab — CHANGES_REQUESTED** [review]
Largely pretty cool but please submit one tool/line item per pull request.

File: README.md [hunk 1]
```diff
@@ -307,6 +318,8 @@ See also [awesome-pentest § Social Engineering Tools](https://github.com/fabaca
 - [APTSimulator](https://github.com/NextronSystems/APTSimulator) - Toolset to make a system look as if it was the victim of an APT attack.
 - [Atomic Red Team](https://atomicredteam.io/) - Library of simple, automatable tests to execute for testing security controls.
 - [BadBlood](https://www.secframe.com/badblood/) - Fills a test (non-production) Windows Domain with data that enables security analysts and engineers to practice using tools to gain an understanding and prescribe to securing Active Directory.
+- [BloodHound](https://github.com/BloodHoundAD/BloodHound) - Open source application used for analyzing security of active directory domains. It performs data ingestion from Active Directory domains and highlights the potential for escalation of rights in Active Directory domains, thus uncovering hidden or complex attack paths that can compromise security of a network.
```
[the remark this answers is not in the document]
Comment (D3vil0p3r) [reply README.md:321]: Ok, so I don't open a PR for it.

File: README.md [hunk 2]
```diff
@@ -307,6 +318,8 @@ See also [awesome-pentest § Social Engineering Tools](https://github.com/fabaca
...
+- [Blue Team Training Toolkit](https://www.bt3.no/) - Defensive security training software that allows you to create realistic attack scenarios with specific IoCs and evasion techniques. You can create training sessions with behavioral and traffic patterns associated with malware, without actually executing real and dangerous malware.
```
[the remark this answers is not in the document]
Comment (D3vil0p3r) [reply README.md:322]: Ok. since it reached EOL, I don't open a PR for it.

File: README.md [hunk 3]
```diff
@@ -381,12 +399,14 @@ See also [awesome-threat-detection](https://github.com/0x4D31/awesome-threat-det
 - [PowerForensics](https://github.com/Invoke-IR/PowerForensics) - All in one PowerShell-based platform to perform live hard disk forensic analysis.
 - [rastrea2r](https://github.com/rastrea2r/rastrea2r) - Multi-platform tool for triaging suspected IOCs on many endpoints simultaneously and that integrates with antivirus consoles.
 - [Redline](https://www.fireeye.com/services/freeware/redline.html) - Freeware endpoint auditing and analysis tool that provides host-based investigative capabilities, offered by FireEye, Inc.
+- [ThreatHunting](https://github.com/olafhartong/ThreatHunting) - Splunk app that contains numerous dashboards and over a hundred reports that will help you enable hunting indicators, allowing you to investigate them further.
```
[the remark this answers is not in the document]
Comment (D3vil0p3r) [reply README.md:402]: Ok. I don't create a PR for it.

**Comment by D3vil0p3r** [comment]
@fabacab Thank you for the answer. According to the comments above, I close this PR since nothing of the tools proposed could be in the scope of the repository.

**Pull request closed without merging**